.github/CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 Whether you are a novice or experienced software developer, all contributions and suggestions are welcome!
 
-Our main contributing guide can be found [in this repo](https://github.com/pandas-dev/pandas/blob/master/doc/source/development/contributing.rst) or [on the website](https://pandas-docs.github.io/pandas-docs-travis/development/contributing.html). If you do not want to read it in its entirety, we will summarize the main ways in which you can contribute and point to relevant sections of that document for further information.
+Our main contributing guide can be found [in this repo](https://github.com/pandas-dev/pandas/blob/master/doc/source/development/contributing.rst) or [on the website](https://dev.pandas.io/docs/development/contributing.html). If you do not want to read it in its entirety, we will summarize the main ways in which you can contribute and point to relevant sections of that document for further information.
```

Comment:
I think should just use pandas.pydata.org but cc @datapythonista

Reply:
I'm trying to move things forward with the hosting, domains... but there are some issues. Will try to find a solution soon (whether it's with OVH as planned, or with the current server, where there are also issues).

I think the idea is to simply use pandas.pydata.org, and stop using pandas.io, dev.pandas.io, pandas-docs.github.io...

There shouldn't be a dev website anymore, because we'll push the master website to production at every commit.

And the dev docs should be at https://pandas.pydata.org/docs/dev/ if I'm not wrong. So, I think this link should be https://pandas.pydata.org/pandas-docs/dev/development/contributing.html or https://pandas.pydata.org/docs/dev/development/contributing.html. @jorisvandenbossche mentioned a preference somewhere, but don't really remember.

Reply:
That was here: https://github.com/pandas-dev/pandas/pull/32530#issuecomment-601795792

Reply:
Can you change this link too? https://pandas.pydata.org/docs/dev/ is already working, and I'm planning to stop using the dev.pandas.io domain soon.

If there is any other reference to pandas.io or dev.pandas.io anywhere else in our docs or repo, would be good to replace it too.

Thanks!